mu-plugins/blocks/table-of-contents/index.php

```diff
@@ -69,8 +71,8 @@ function render( $attributes, $content, $block ) {
 	// Use the parsed headings & IDs to inject IDs into the post content.
 	add_filter(
 		'the_content',
-		function( $content ) use ( $items ) {
-			return inject_ids_into_headings( $content, $items );
+		function() use ( $items, $post_content ) {
+			return inject_ids_into_headings( $post_content, $items );
```

Comment:
This one should remain unchanged, as we want to operate on the $content that's being passed through the filter, not what we retrieved earlier.

Reply:
Hmm, it wasn't returning the content properly without this change.

Reply:
Probably because of the priority — this had to be priority 5, but your update to the content in https://github.com/WordPress/wporg-developer/pull/194 happens later, at prio 10. If I switch the filter priority in 194 to `4`, it works.

Reply:
Ah that makes sense. I'll revert this out.